src/renderer/services/thorchain/types.ts

```diff
@@ -84,16 +84,19 @@ export type GetLiquidityProvidersParams = {
 }
 
 export type GetLiquidityProviderParams = GetLiquidityProvidersParams & {
-  runeAddress: O.Option<Address>
-  assetAddress: O.Option<Address>
+  runeAddress: Address
```

Comment:
i would keep `runeAddress: O.Option<Address>`:
- `assetAddress` is always required for both `sym`/`asym` providers
- `runeAddress` is optional to detect if we want to get `sym`/`asym` provider.
i know we now "ignore" all asym-related stuff but if this is a brand-new stuff we should be "aimed" to the future stuff

Reply:
Good points as well!

Reply:
_Will be done with another PR - see #1512_

Reply:
> runeAddress is optional to detect if we want to get sym/asym provider.

That's not correct, but as follow: "if member is assym - they will only have either rune_address or asset_address in their member details." (see https://thorchain.slack.com/archives/C012CLYR1DG/p1612347969232600?thread_ts=1612288793.231200&cid=C012CLYR1DG). We will check always both addresses then.

Check for pending asset is for sym. deposit needed only. If an asym. deposit might fail, there won't be a pending asset.